src/debugCodeLensProvider.ts

```diff
@@ -166,6 +166,11 @@ async function constructDebugConfig(mainClass: string, projectName: string, work
     return _.cloneDeep(debugConfig);
 }
 
+async function launchJsonExists(): Promise<boolean> {
+    const results: vscode.Uri[] = await vscode.workspace.findFiles(".vscode/launch.json", null, 1);
```

Comment:
Will this work in multi-root? How about `**/.vscode/launch.json`?

Reply:
VS Code will search from each workspace root. This path is the path with the workspace root.

`**/.vscode/launch.json` will match `myProject/.settings/.vscode/launch.json` file, actually vscode will not take it as a launch.json.